Broken in naninovel 1.19

When included in a project that uses naninovel 1.19 (instead of 1.18, which this is built for), this error breaks the project:
```
Library\PackageCache\net.bobbo.sgb.imod.naninovel@1.4.0\Runtime\FrontendModeManager.cs(24,76): error CS1061: 'IScriptManager' does not contain a definition for 'StartGameScriptName' and no accessible extension method 'StartGameScriptName' accepting a first argument of type 'IScriptManager' could be found (are you missing a using directive or an assembly reference?)
```

--- a/Runtime/FrontendModeManager.cs
+++ b/Runtime/FrontendModeManager.cs
@@ -21,7 +21,7 @@ namespace BobboNet.SGB.IMod.Naninovel
                 // If there IS no return script setup, default to the engine's starting script
                 if (string.IsNullOrWhiteSpace(returnScriptName))
                 {
-                    returnScriptName = Engine.GetService<IScriptManager>().StartGameScriptName;
+                    returnScriptName = Engine.GetConfiguration<ScriptsConfiguration>().StartGameScript;
                 }
 
                 // Load back into the desired return script in naninovel

--- a/package.json
+++ b/package.json
@@ -8,6 +8,6 @@
   },
   "dependencies": {
     "com.unity.inputsystem": "1.2.0",
-    "net.bobbo.sgb.imod": "1.4.0"
+    "net.bobbo.sgb.imod": "1.5.0"
   }
 }

